content/en/snippets/others/select_text.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "put text in a selecting state"
+weight: 10
+ie_support: true
+---
+
+This snippet is used to put the text of an element in a selection state.
+Change the following value.
+
+- `targetSelector`: A string of a selector to specify the element
+
+```js
+var targetSelector = "<TODO: REPLACE SELECTOR>";
+
+/* --------- Anything below this does not need to be changed. ---------- */
+var element = document.querySelector(targetSelector);
+
+if (!element) {
+	throw new Error("Element not found");
+}
+
+/* Select the text of an element */
+selectRange(element, 0, element.textContent.length);
+
+function selectRange(element, startAt, endAt) {
```

Comment:
Changing the argument name `element` would be better because `element` is already used outside this function.
We know JS uses `element` from arguments but it will prevent readers from confusing.

Reply:
@otoyo 
I have fixed the comments I received and a point.
I want to generalize the naming of function arguments, so I changed the variable names.
Could you please check again?m(_ _)m